### ISSUE TITLE
weird results for custom images

### ISSUE THREAD

**Issue by jjxxmiin** [ZPdesu/SEAN#32]
Thanks for providing good code.

I want to use a custom image with pretrained model.

But when I create the original custom image with the custom image and mask it gives weird results.

![image](https://user-images.githubusercontent.com/39845657/120270948-5be58580-c2e5-11eb-850c-b75df07a6df4.png)

![image](https://user-images.githubusercontent.com/39845657/120270963-6142d000-c2e5-11eb-9fef-5c04980e5c6c.png)

It produces accurate results when using celebAMask images, but not accurate results when using custom images.

How can I solve this?

Note: I used the Bisenet model to generate the mask. -> [Here](https://github.com/zllrunning/face-parsing.PyTorch)
